server.R

```diff
@@ -378,11 +371,23 @@ function(input, output, session) {
     }
   )
   
+  output$downloadImpermissibleCharacterDetailBtn <- renderUI({
+    req(nrow(file_structure_analysis_main()) > 0)
```

Comment:
I tried running this with a "good" file and I'm seeing the button once it successfully gets uploaded. When I click the button, it says "Site wasn't available"